datahub-web-react/src/app/shared/numberWithCommas.tsx

```diff
@@ -0,0 +1,4 @@
+// Function to convert a number with comma
+export const numberWithCommas = (number) => {
```

Comment:
We don't need this. We already have a "formatNumber" method that we use everywhere else! Please just reuse that

Reply:
Deleted **numberWithCommas** function and used **formatNumber** to format number.